packaging/fedora/snapd.spec

```diff
@@ -731,6 +731,9 @@ popd
 %{_unitdir}/snapd.autoimport.service
 %{_unitdir}/snapd.failure.service
 %{_unitdir}/snapd.seeded.service
+%{_userunitdir}/snapd.session-agent.service
+%{_userunitdir}/snapd.session-agent.socket
+%{_userunitdir}/sockets.target.wants/snapd.session-agent.socket
```

Comment:
This is not allowed in Fedora. We need to get the user-preset added in Fedora for this.